draft-specifications/CapTP Specification.md

```diff
@@ -419,21 +437,22 @@ An example of how to use this method is:
 
 ## `deposit-gift` Method
 
-The deposit gift method is used in conjunction with sending a [Third Party
-Handoff](#third-party-handoffs). This method is used to deposit a gift which has
-been sent to the bootstrap object. It has two arguments:
+The deposit gift method is used in conjunction with sending a [Third Party Handoff](#third-party-handoffs).
+This method is used to deposit a gift which has been sent to the bootstrap object. It has two arguments:
 
 1.  A gift ID that is non-negative integer.
-2.  A `desc:import-object` or `desc:import-promise` which has been exported
+2.  A [Reference][Model-Reference] which has been exported
     within the given CapTP session.
 
+This should have been sent with the [`op:deliver-only`](#op-deliver-only) operation.
+
 Here is an example of how to use this method:
 
 ```text
-<op:deliver-only <desc:export 0>                               ; Remote bootstrap object
-                 ['deposit-gift                                ; Argument 1: Symbol "deposit-gift"
-                  gift-id                                      ; Argument 2: Non-negative integer (>=0)
-                  <desc:import-object | desc:import-promise>]> ; Argument 3
+<op:deliver-only <desc:export 0>         ; Remote bootstrap object
+                 ['deposit-gift          ; Argument 1: Symbol "deposit-gift"
+                  gift-id                ; Argument 2: Non-negative integer (>=0)
+                  <desc:import-object]>  ; Argument 3
```

Comment:
This is missing the `>` for <desc:import-object>, it also probably wants to include a specific imported object e.g. `<desc:import-object 5>` or something.

I think it's fine if we don't, but while we're improving the spec's wording I wonder if we should include a description of this argument, e.g. `Argument 3: object being shared via handoff`.